index.html

```diff
@@ -102,95 +110,129 @@ <h2>SIMPLE AND BEAUTIFUL</h2>
                     </p>
                 </di>
             </section>
-            <section class="container__pricing" id="pricing">
+            <header class="container__pricing" id="pricing">
                 <div class="pricing__header">
                     <h2>CHOOSE YOUR PRICING PLAN</h2>
                     <p>Pick any of our super affordable pricing plans</p>
                 </div>
                 <div class="pricing__header-items">
-                    <div class="pricing__item-first">
+                    <div class="pricing__item">
                         <h3>BASIC</h3>
                         <h2 class="pricing__header-price">$15</h2>
-                        <p>per month</p>
-                        <p>Up to 7 Project 2 Additional Developers</p>
+                        <ul class="pricing__header-month">
+                            per month
+                        </ul>
+                        <li class="pricing__list">
+                            Up to 7 Project 2 Additional Developers
+                        </li>
```

Comment:
Tutaj powinno być coś takiegego:
```
<p>per month</p>
<ul>
   <li>up to 7 project</li>
   <li>additionl developers</li>
 </ul>
```

Wtedy będzie "wypunktowanie" (każdy punkt w nowej linii).

Obecnie jest to ciągiem pisane:
![image](https://github.com/devmentor-pl/task-html-and-css-basics/assets/55095257/2df22dd7-727c-4c37-90b7-c88f578f53a2)